utils/mstfwmanager/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2024
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mstfwmanager
+PKG_VERSION:=4.26.0-1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/Mellanox/mstflint
+PKG_MIRROR_HASH:=2592515184fe229690f2567ee42b5e2a0c847c14a2d8924175384655caf649e7
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE_DATE:=2023-11-08
```

Comment:
Can not we use tagged releases? Is there something why do you need to use Git sources?